src/createSchemaForm.tsx

```diff
@@ -141,6 +141,143 @@ function propsMapToObect(propsMap: PropsMapping) {
   return r;
 }
 
+type SchemaShape<SchemaType extends RTFSupportedZodTypes | ZodEffects<any, any>> = ReturnType<UnwrapEffects<SchemaType>['_def']["shape"]>
```

Comment:
wanted to ask you if we really need the '_def' here. i think shape is exposed on the top level type and it seemed to work for me but wasn't sure if this was on purpose.

Reply:
I think it's the same so shouldn't matter